code/modules/clothing/suits/armor.dm

```diff
@@ -78,17 +78,31 @@
 
 /obj/item/clothing/suit/armor/riot
 	name = "riot suit"
-	desc = "A suit of armor with heavy padding to protect against melee attacks. Looks like it might impair movement."
+	desc = "A suit of armor with heavy padding to protect against melee attacks."
 	icon_state = "riot"
 	item_state = "swat_suit"
 	body_parts_covered = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	cold_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
 	heat_protection = CHEST|GROIN|LEGS|FEET|ARMS|HANDS
-	armor = list(melee = 50, bullet = 10, laser = 10, energy = 10, bomb = 0, bio = 0, rad = 0)
+	armor = list(melee = 70, bullet = 10, laser = 10, energy = 10, bomb = 0, bio = 0, rad = 0)
 	flags_inv = HIDEJUMPSUIT
 	strip_delay = 80
 	put_on_delay = 60
 
+/obj/item/clothing/suit/armor/eod
+	name = "EOD suit"
```

Comment:
From the PR the EOD is really more of a port of Goon's EOD suit. You raise a good point and a reminder about the Bomb Suit as well as the Sec Bio Suit.

A name change would be fine, Bulldozer is not exactly being subtle however.

I'll address the EOD and Bomb Suit imbalance.